resources/kcp/charts/kyma-environment-broker/values.yaml

```diff
@@ -281,3 +281,14 @@ serviceMonitor:
 
 dashboardConfig:
   landscapeURL: "https://dashboard.dev.kyma.cloud.sap"
+
+setupJobs:
+  - name: kcp-kyma-environments-cleanup
+    schedule: "0 0 * * *"
+    namespace: kcp-system
+    imageName: kyma-environments-cleanup
+    cronJobName: environments-cleanup
+    maxAge: "24h"
+    labelSelector: "owner.do-not-delete!=true"
```

Comment:
nit: I couldn't find any reference of this string prior to this PR, is the content intentional?